ROADMAP.md

```diff
@@ -113,6 +131,7 @@ The list is sorted by priority
     * Buddy allocator for sender data-segment
     * Introduce runtime fixed-size types
 * [x] Untyped API
+* [ ] Zero-copy based on dmaheap in linux
```

Comment:
Only if you have to create another meaningful commit, else this can be done later. Could you add a dash to dmaheap to make it easier to read, like dma-heap?